src/App.js

```diff
@@ -1,5 +1,40 @@
+import { Route, Routes } from "react-router-dom";
+import { Sidebar, Header } from "./components";
+import {
+  History,
+  Homepage,
+  Liked,
+  Playlist,
+  Videos,
+  WatchLater,
+} from "./pages";
+
 function App() {
-  return <h1>CobraTube</h1>;
+  return (
+    <div>
+      <Header />
+      <div className="app">
+        <Sidebar />
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          {/* <Route element={<AuthRoutes />}>
+          <Route path="/login" element={<Login />} />
+          <Route path="/signup" element={<Signup />} />
+        </Route> */}
+          {/* <Route element={<RequiresAuth />}> */}
```

Comment:
remove comments if not intended for future use

Reply:
These are for future use